mobile/services/queries/notes.query.ts

```diff
@@ -32,7 +32,9 @@ export const useNotesForQuestionId = (
     formId,
     useCallback(
       (data: Note[]) => {
-        return data.filter((note) => note.questionId === questionId);
+        return data
+          .filter((note) => note.questionId === questionId)
+          .sort((a, b) => +new Date(b.createdAt) - +new Date(a.createdAt)); // added unary '+' operator to avoid ts error that the right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type
```

Comment:
i did the sorting here in order to display the notes in DESC order, but would it be wiser to ask the BE to send them already sorted?

Reply:
It's ok for now, should be a small array